.gitignore

```diff
@@ -2,6 +2,8 @@
 
 # build tools
 /node_modules
+package-lock.json
```

Comment:
We probably do want a lock file checked in.... although we typically use `yarn` these days. 🤔  Not trying to rabbit hole this PR.  Maybe we should be checking in the `yarn.lock` in that case and leaving this `package-lock.json` in `.gitignore` to signal that folks shouldn't try to check that in.

Reply:
I was just trying not to add any unnecessary changes. I had a sense that this was built with `npm` because of its age, so went with that and made sure not to merge anything not necessary to the changes I was looking for. I can remove my changes to this file.

Reply:
It would have made more sense just to not commit those files 😆